source/includes/_endpoint-filter.md

```diff
@@ -14,7 +14,8 @@ This index contains two kinds of filters: public and private, denoted by the `is
     "name": "My filter",
     "is_public": true,
     "id": "1442ea",
-    "owner_id": "https://app.crunch.io/api/users/4152de/"
+    "owner_id": "https://app.crunch.io/api/users/4152de/",
```

Comment:
I thought we were planning to stop using full URL's for things like this?

Reply:
I was maintaining the usage on the file not to have it look different in the same explanation.